dt-core/utilities/dt-components.php

```diff
@@ -114,6 +114,26 @@ public static function render_date( $field_key, $fields, $post, $params = [] ) {
         <?php
     }
 
+    public static function render_datetime( $field_key, $fields, $post, $params = [] ) {
+        $shared_attributes = self::shared_attributes( $field_key, $fields, $post, $params );
+        ?>
+        <dt-datetime <?php echo wp_kses_post( $shared_attributes ) ?>
+            timestamp="<?php echo esc_html( $post[$field_key]['timestamp'] ?? '' ) ?>">
+            <?php dt_render_icon_slot( $fields[$field_key] ) ?>
+        </dt-datetime>
+        <?php
+    }
+
+    public static function render_number( $field_key, $fields, $post, $params = [] ) {
+        $shared_attributes = self::shared_attributes( $field_key, $fields, $post, $params );
+        ?>
+        <dt-number <?php echo wp_kses_post( $shared_attributes ) ?>
+            value="<?php echo esc_html( $post[$field_key] ?? '' ) ?>">
+            <?php dt_render_icon_slot( $fields[$field_key] ) ?>
+        </dt-number>
```

Comment:
The number field also can have min/max values. See https://developers.disciple.tools/theme-core/customization/fields for the syntax in PHP and https://jade-chebakia-17493f.netlify.app/?path=/story/components-form-number--min-max-value for the component attributes.